Jint/Runtime/EventLoop.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+using System.Collections.Generic;
+
+namespace Jint.Runtime
+{
+    public sealed class EventLoop
```

Comment:
internal?

Reply:
Likely, as of right now it is a return value of `ExecuteWithEventLoop` method, I'm debating to not expose that at all. Tentatively it will be moved to internal if I come up with ergonomic API 

If I can get away with the following I will make it internal
```csharp 
public Engine ExecuteWithEventLoop(
   Script script,
   Action onFinished
);
```

Reply:
`Execute` methods are there to allow chaining, I think promise execution would get away with the `JsValue EvaluateXXX` kind of method as it's all evaluation without (hopefully) need to get engine instance in chain call

Reply:
Marked EventLoop as internal. It is no longer exposed